service/service.go

```diff
@@ -146,3 +146,19 @@ func New(options ...Option) (Owner, error) {
 
 	return svc, nil
 }
+
+type niladicStart func()
```

Comment:
It's not a noop though. A noop is a stub, this is a function with zero arguments that **does** get called.